UefiPayloadPkg/SPI/pci_ops.c

```diff
@@ -0,0 +1,52 @@
+// #include <Include/PiDxe.h>
```

Comment:
Use PciLib from EDK2 and delete this file

Reply:
@IgorBagnucki same as above

Reply:
Done